File: dataset/human/c/file_2949.c
#include <stdio.h>
int main()
{
	int K, S, counter;
	scanf("%d %d", &K, &S);
	counter = 0;

	for(int x = 0; x <= K; ++x){
		for(int y = 0; y <= K; ++y){
			int z = S - (x + y);
			if((z >= 0) && (z <= K))
				++counter;
		}
	}
	printf("%d\n", counter);
	return 0;
}

File: dataset/human/python/file_2949.py
import collections

def trial_division(n):
    if n < 2:
        return []
    
    prime_factors = []

    for i in range(2, int(n**0.5)+1):
        while n % i == 0:
            prime_factors.append(i)
            n //= i
    
    if n > 1:
        prime_factors.append(n)
    
    return prime_factors

counter = 0
if __name__ == "__main__":
    N = int(input())
    
    c = collections.Counter(trial_division(N))
    for primary_number, count in c.items():
        for i in range(1, count+1):            
            z = primary_number ** i
            if N % z == 0:
                counter += 1
                N //= z
    
    print(counter)
